src-docs/src/views/home/home_illustration.tsx

```diff
@@ -9,20 +9,18 @@
  * GitHub history for details.
  */
 
-import React, { useContext } from 'react';
-import { ThemeContext } from '../../components/with_theme';
-import illustrationDarkMode from '../../images/illustration-oui-hero-500-darkmode-shadow.svg';
-import illustrationLightMode from '../../images/illustration-oui-hero-500-shadow.svg';
-import { OuiImage } from '../../../../src/components/image';
+import React from 'react';
+//import { ThemeContext } from '../../components/with_theme';
+//import { OuiImage } from '../../../../src/components/image';
 
 function Icon() {
-  const themeContext: any = useContext(ThemeContext);
+  //const themeContext: any = useContext(ThemeContext);
```

Comment:
Honestly thinking we should create a follow up issue to add a logo in the view if we want and delete this un used code.

Or delete this file.

Th

Reply:
Okay will change

Reply:
I do not want to create a follow up issue to add a graphic here. I am being very judicious with how much time we spend on this forked site in favor of moving over to the storybook solution in the future. Lets move forward with delete.